docs/README.md

```diff
@@ -75,6 +75,16 @@ this for you, but you must first install [`mkcert`](https://github.com/FiloSotti
 ./hack/kind/kind.sh cacert
 ```
 
+In order to join a self-signed Apex controller from a remote node or view the Apex UI in your dev environment, you will need to install the cert on the remote machine. This is only necessary when the controller is self-signed with a domain like we are using with the apex.local domain for development.
+
+Install [`mkcert`](https://github.com/FiloSottile/mkcert) on the agent node, copy the cert from the controller running kind (`.certs/rootCA.pem`) to the remote node you will be joining (or viewing the web UI) and run the following. 
+
+```console
+CAROOT=$(pwd)/.certs mkcert -install
+```
+
+For windows, we recommend installing the root certificate via the MMC snap-in.
```

Comment:
Would windows users know what this means?  or could it use a link for more info?